sscanss/app/dialogs/managers.py

```diff
@@ -943,3 +947,153 @@ def moveDetectorsButtonClicked(self):
         self.parent.scenes.changeVisibility(Attributes.Beam, False)
         event.accept()
+
+
+class SampleProperties(QtWidgets.QWidget):
+    """Creates a UI for viewing sample properties
+
+    :param parent: main window instance
+    :type parent: MainWindow
+    """
+    dock_flag = DockFlag.Bottom
+
+    def __init__(self, parent):
+        super().__init__(parent)
+        self.parent = parent
+        self.parent_model = parent.presenter.model
+
+        self.main_layout = QtWidgets.QVBoxLayout()
+        self.sample_property_table = QtWidgets.QTableWidget()
+        self.sample_property_table.horizontalHeader().setSectionResizeMode(QtWidgets.QHeaderView.Stretch)
+        self.sample_property_table.verticalHeader().hide()
+        self.sample_property_table.setColumnCount(2)
+        self.sample_property_table.setRowCount(5)
+        self.sample_property_table.setAlternatingRowColors(True)
+        self.sample_property_table.setMinimumHeight(165)
+        self.sample_property_table.setMaximumHeight(210)
+        stylesheet = "::section{Background-color:rgb(100,100,100); font-weight: bold; color: rgb(255, 255, 255);}"
+        self.sample_property_table.horizontalHeader().setStyleSheet(stylesheet)
+        self.sample_property_table.setHorizontalHeaderLabels(['Property', 'Value'])
+        self.sample_property_table.setEditTriggers(QtWidgets.QAbstractItemView.NoEditTriggers)
+        self.addSamplePropertyLabels()
+        self.addSamplePropertyValues()
+        self.sample_property_table.resizeRowsToContents()
+        self.main_layout.addWidget(self.sample_property_table, 20)
+        self.main_layout.addStretch(1)
+        self.setLayout(self.main_layout)
+        self.title = 'Sample Properties'
+        self.setMinimumWidth(450)
+        self.parent_model.sample_changed.connect(self.updateTable)
+
+    def updateTable(self):
+        """Updates the contents of the table when sample changes"""
+        self.sample_property_table.clearContents()
+        self.addSamplePropertyLabels()
+        self.addSamplePropertyValues()
+
+    def addSamplePropertyLabels(self):
+        """Adds the sample property labels to the table"""
+        self._addLabels({0: 'Memory (MB)'})
+        if isinstance(self.parent_model.sample, Mesh):
+            self._addLabels({1: 'Faces', 2: 'Vertices'})
+        elif isinstance(self.parent_model.sample, Volume):
+            self._addLabels({1: 'Dimension', 2: 'Voxel size', 3: 'Minimum Intensity', 4: 'Maximum Intensity'})
+
+    def _addLabels(self, label_dict):
+        """Adds the property label to the table
+
+        :param label_dict: the dictionary containing the property labels
+        :type label_dict: dict
+        """
+        for row, label_tag in label_dict.items():
+            self._add_property_label(label_tag, row=row)
+
+    def _addMeshPropertyValues(self):
+        """Adds the mesh property values to the table"""
+        memory_str = self._getMemoryUsed()
```

Comment:
Unnecessary variable. _getMemoryUsed() is already very clear

Reply:
Unnecessary variable removed.